### DISCUSSION TITLE
I can't even see the status of my application... It's bugged I think

### DISCUSSION THREAD

**Discussion by lucasordenes** [community/community#116549] [GitHub Education]
### Hi everyone,

I can't even see the status of my application, I have submitted like 5 times, even tried to submit completely different proof, now I get the response of I already submitted that, either that or nothing, not even an error...

I click see status of application and it keeps asking for my stuff... I don't get any confirmation email of upload, nothing

edit: forgot to add that I tried a few days ago and today, still nothing.
I upload an get this
![image](https://github.com/community/community/assets/152818189/15569b82-c887-4026-a8e2-d78dd926ba9f)
or nothing and back to asking for proof

**Comment by samus-aran** [comment]
Hi @lucasordenes 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍

**Reply by lucasordenes** [reply]
Thanks I had to take a picture of the exact same thing I was sending but with the camera of my phone and now it worked.... But then why It's even an option to upload a file, that sucks a lot.
